src/resource/mock.test.ts

```diff
@@ -133,14 +137,31 @@ describe("mockFetchError", () => {
   });
 
   it("can represent unknown status codes", () => {
-    const error = mockFetchError("https://some.pod/resource", 1337);
+    const error = mockFetchError(
+      "https://some.pod/resource",
+      599,
+      "Unknown error"
+    );
 
     expect(error).toBeInstanceOf(Error);
-    expect(error.statusText).toBeUndefined();
-    expect(error.message).toBe(
-      "Fetching the Resource at [https://some.pod/resource] failed: [1337] [undefined]."
+    expect(error.statusCode).toBe(599);
```

Comment:
Why change the error code - seems kind of arbitrary?

Reply:
That was a fun one: as per https://httpwg.org/specs/rfc9110.html#rfc.section.15, status codes go from 100 to 599. The former `Response` constructor didn't complain, but the new one did.